src/preamble.js

```diff
@@ -472,9 +472,6 @@ function abort(what) {
   /** @suppress {checkTypes} */
   var e = new WebAssembly.RuntimeError(what);
 
-#if MODULARIZE
-  readyPromiseReject(e);
-#endif
```

Comment:
Why is this no longer needed?

Reply:
I've marked this is a draft for now.  I'll come back to it in the new year .